helix-term/src/ui/editor.rs

```diff
@@ -337,48 +332,71 @@ impl EditorView {
         doc: &Document,
         view: &View,
         theme: &Theme,
-        cursor_shape_config: &CursorShapeConfig,
+        config: &DynGuard<Config>,
```

Comment:
```suggestion
        config: &helix_view::editor::Config,
```

see `render_text_highlights` which also takes config. This will eliminate the new imports.